scripts/validate-conventional-commit-prefix.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+code=0
+while read -r commit; do
+    match=$(echo "${commit}" | grep -o -h -E "^[a-z]+(\([a-z]+\))?: " || true)
+    if [[ -z "${match}" ]]; then
+        echo "[ ] Commit \"${commit}\" does not have the required conventional commit prefix."
+        code=1
+    else
+        echo "[x] Commit \"${commit}\" has conventional commit prefix \"${match}\"."
+    fi
+done < <(git --no-pager log --pretty=format:%s && echo "") # git log does not include newline after last commit
+
+exit ${code}
```

Comment:
Imo very useful feature and +1 to declare a canonical form for the commits

Just a question: would it make sense to mention https://www.conventionalcommits.org/en/v1.0.0/#summary also in the bash script? folks who are not familiar with _conventional commit_  can be redirected to the source from CI directly (when the exit code is other than 0)
Or mb a ref to ```CONTRIBUTING.md#code-contributions```

Reply:
Makes sense, I added a link and used the GitHub Actions error format to bring the CI error to the surface.